src/client/helpers/helpers.js

```diff
@@ -287,6 +296,21 @@ function deletePendingMemories(cb) {
     cb();
   })
 }
+if(localStorage.getItem('textToAdd') !== null) {
+const newTextMemories = JSON.parse(localStorage.getItem('textToAdd'));
+newTextMemories.memories.forEach((memory, index) => {
+  const text = memory.text;
+  const heading = memory.heading;
+  const tag = memory.tag;
+  $.ajax({
+    method: 'POST',
+    url: 'memory-input-text',
+    data: {memory_text: text, heading, tag},
+    success: () => clearPendingActions('textToAdd', index),
+  });
+  cb();
```

Comment:
😭  - it's rerendering after each update so if there are several updates, there are several transitions. Is there a better way to do it?